Tools/Scripts/libraries/webkitcorepy/webkitcorepy/terminal.py

```diff
@@ -57,11 +57,13 @@ def choose(cls, prompt, options=None, default=None, strict=False, numbered=False
                     ('{}) [{}]' if options[i] == default else '{}) {}').format( i + 1, options[i])
                     for i in range(len(options))
                 ]
-                response = cls.input('{}:\n    {}\n: '.format(prompt, '\n    '.join(numbered_options)))
+                sys.stdout.write('{}:\n    {}\n'.format(prompt, '\n    '.join(numbered_options)))
             else:
-                response = cls.input('{} ({}): '.format(prompt, '/'.join([
```

Comment:
Should we consider fix in `input` method as I can imaging there could be other places will have the same issue?

Reply:
The trouble with fixing `cls.input` is that it's basically just `raw_input`/`input` with a try/catch. I'm reluctant to make `input` mean anything other than Python's native `input` (even if we agree that Python's native `input` function has issues)